### PR TITLE
Variable pass

### PR DESCRIPTION
**Pull request by guswadsworth12**
Made a pass through and swapped out colors for variables. 

Standarized the item sheet through the use of .armor css class. 

Testing
Please verifying the styling of ALL item sheets.

### CODE REVIEW

File: sass/items/_armor.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  background: v.$background-b;
+  border: v.$border;
+  border-radius: v.$border-radius;
+  color: v.$textcolor1;
+  padding: v.$padding;
+}
+
+.essence20 .sheet-body{
+  min-height: auto;
+}
```
Comment (phildominguez) [inline sass/items/_armor.scss:13]: Any objections to putting these in _item-styling.scss and changing `.armor` to `.item-sheet` or something? It's confusing to me to use armor everywhere.

**Comment by phildominguez** [comment]
Is it possible to make the item sheet min height fit the content? For example, I can't resize a Specialization to be any shorter than this:
image

**Comment by phildominguez** [comment]
For some reason the description section is a lot bigger than the actual editor
image

**Review by phildominguez — CHANGES_REQUESTED** [review]
Looks awesome! Left a few comments. I'm gonna make a task to flexify the Origin and Resource sheets as well, no need to do that here.

File: sass/items/_armor.scss (hunk 1 above)
Comment (guswadsworth12) [reply sass/items/_armor.scss:13]: not at all. Ill do that now

Comment (guswadsworth12) [reply sass/items/_armor.scss]: Swapped out now

**Review by phildominguez — APPROVED** [review]

**Pull request merged**